### ISSUE TITLE
Open/Tabopen Not Using Location Bar Entries

### ISSUE THREAD

**Issue by rmegal** [vimperator/vimperator-labs#596]
##### Issue type:

- Bug report

##### Version:

```
Vimperator:	3.14.0 (created: 2016/06/07 23:58:16)
Firefox:	Mozilla/5.0 (Windows NT 10.0; WOW64; rv:49.0) Gecko/20100101 Firefox/49.0
```

##### Description:

The Firefox Location Bar entries (e.g. history) no longer show up when I use the open or tabopen commands. I've disabled/enabled and uninstalled/installed vimperator without effect. CPT is the default of sl. Setting it to slh does force history to show up. The Location bar seems to work as expected if I use Ctrl-L to set focus there.

##### Expected behavior:

The open and tabopen commands should present the Location bar contents when CPT is set to sl.

##### Steps to reproduce:

Set cpt to sl and the Location bar entries no longer show up in the open and tabopen commands.

**Comment by albertdev** [comment]
This might be a duplicate of #590 and #594, it might be fixed in pull request #591.

**Comment by rmegal** [comment]
I agree. This is a duplicate.
